qiime_studio/api/jobs.py

```diff
@@ -74,6 +74,15 @@ def create_job():
             parameters[key] = qiime.Metadata.load(
                 parameters[key][0]).get_category(parameters[key][1])
             json_params[key] = '<metadata>'
+        elif type_ == qiime.plugin.Bool:
+            # web forms exclude unchecked values, so we have to artificially
+            # "fill in the blanks" for the missing boolean parameters
+            try:
+                checked = parameters[key]
+            except:
+                checked = 'off'
+            parameters[key] = ('%s' % (checked == 'on')).lower()
+            json_params[key] = parameters[key]
```

Comment:
It could be handled client-side, but I just figured it was easier to handle it in this loop that has all of the signature parameters and semantic types to cross-check, as unchecked items don't get passed at all with the form data, so we could swap the `'on'` for `true` JS side, but would also have to iterate through all parameters again at the end of that loop to see which ones are missing and would have to get set to `false`. I have no strong feelings either way though.